test/e2e_node/util.go

```diff
@@ -45,13 +45,15 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
 // TODO(random-liu): Get this automatically from kubelet flag.
 var kubeletAddress = flag.String("kubelet-address", "http://127.0.0.1:10255", "Host and port of the kubelet")
 
 var startServices = flag.Bool("start-services", true, "If true, start local node services")
 var stopServices = flag.Bool("stop-services", true, "If true, stop local node services after running tests")
+var BusyboxImage = imageutils.GetBusyBoxImage()
```

Comment:
nit: does this need to be exported? i.e. should it instead be `busyBoxImage`?

Reply:
Fixed it.